Marlin/planner.h

```diff
@@ -393,7 +393,7 @@ class Planner {
 
     #if ENABLED(ENSURE_SMOOTH_MOVES)
       static bool long_move() {
-          if (blocks_queued()) {
+          if (blocks_queued() && block_buffer_runtime_us) {
```

Comment:
Tiny performance note. Whichever test runs faster should be in front of the `&&`.